tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp

```diff
@@ -79,6 +80,17 @@ void test_schwarzschild() {
   Approx custom_approx = Approx::custom().epsilon(1.e-12).scale(1.0);
   CHECK_ITERABLE_CUSTOM_APPROX(
       get(residual), DataVector(get(residual).size(), 0.0), custom_approx);
+
+  const auto ricci_scalar = StrahlkorperGr::ricci_scalar(
+      make_spatial_ricci_schwarzschild(cart_coords, mass),
+      unit_normal_vector,
+      StrahlkorperGr::extrinsic_curvature(grad_unit_normal_one_form,
+                                          unit_normal_one_form,
+                                          inverse_spatial_metric),
+      inverse_spatial_metric);
+  CHECK_ITERABLE_CUSTOM_APPROX(
+      get(ricci_scalar), DataVector(get(ricci_scalar).size(), 0.5/(mass*mass)),
+      custom_approx);
```

Comment:
do you need the custom_approx?

Reply:
No, but I have to lower l=10 to l=9 on the Strahlkorper or there's too much roundoff error for CHECK_ITERABLE_APPROX.